src/scripts/dataframes/unmapped_ordo_lex.tsv

```diff
@@ -0,0 +1 @@
+subject_id	subject_label	predicate_id	object_id	mapping_justification	object_label	comment
```

Comment:
What happened here? Lexmatch didnt find any ORDO mappings? Maybe not using the correct prefix (`Orphanet`)?

Reply:
I was going to ask you about it too. There aren't for ORDO or OMIM. I need to dog deeper

Reply:
first check if the index file contains ORDO/Orphanet classes, if so, it will be some confusion about the prefixes in the analysis code?